File: DDM/09.27/flutter_application_exeapilocalhost/lib/aluno.dart
class Aluno {

  int _ra = 0;
  String _nome = "";

  Aluno();

  Aluno.fromJson(Map <String, dynamic> json)
  :
    _ra = json['ra'],
    _nome = json['nome']
  ;

  int get ra => this._ra;

  set ra(int value) => this._ra = value;

  get nome => this._nome;

  set nome( value) => this._nome = value;

}

File: DDM/09.27/demoApi/src/main/java/com/example/demoApi/model/Aluno.java
package com.example.demoApi.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table (name = "aluno")
public class Aluno {

    @Id
    private int ra;

    @Column
    private String nome;

    public int getRa() {
        return ra;
    }

    public void setRa(int ra) {
        this.ra = ra;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

}
